### PR TITLE
Reactnative/image fixes

### PR DESCRIPTION
**Pull request by Surezt22**
This PR includes fixes for #46 & #47.

As per iOS implementation, for an image set, when the size is undefined or auto, the size of the images are considered as medium.
Followed the same implementation in iOS.

### CODE REVIEW

File: source/reactnative/src/components/elements/image.js [hunk 1]
```diff
@@ -64,7 +66,9 @@ export class Img extends React.Component {
 			Enums.Spacing.Small);
 		this.spacing = this.hostConfig.getEffectiveSpacing(spacingValue);
 		this.separator = this.payload.separator || false;
-		this.backgroundColor = this.payload.backgroundColor || Constants.TransparentString;
+		//[ST]- Fix for the image background color issue
+		this.backgroundColor = Utils.hexToRGB(this.payload.backgroundColor) || Constants.TransparentString;
+		console.log("BG Color:",this.backgroundColor);
```
Comment (BalajiR) [inline source/reactnative/src/components/elements/image.js]: Remove console logs.

File: source/reactnative/src/components/elements/image.js [hunk 2]
```diff
@@ -200,12 +204,29 @@ export class Img extends React.Component {
...
 				default:
 					{
-						sizeStyle.push([styles.imageAuto, {
-							width: this.state.imageWidth,
-							height: this.state.imageHeight
-						}]);
-						this.width = this.state.imageWidth;
-						this.height = this.state.imageHeight;
+									//[ST]- Fix for the imageSet issues, when the size is undefined
+						if ((this.isSizeUndefined && this.payload.fromImageSet == true) ||
```
Comment (BalajiR) [inline source/reactnative/src/components/elements/image.js]: Add comment on if / else cases in 'default'

File: source/reactnative/src/components/elements/image.js [hunk 3]
```diff
@@ -200,12 +204,29 @@ export class Img extends React.Component {
 					}
```
Comment (BalajiR) [inline source/reactnative/src/components/elements/image.js]: Assign 'width' value for 'small' size similar to other categories.

File: source/reactnative/src/components/elements/image.js [hunk 4]
```diff
@@ -51,6 +51,8 @@ export class Img extends React.Component {
 		this.horizontalAlignment = this.getImageAlignment();
 		this.selectAction = this.payload.selectAction || null;
 		if (Utils.isNullOrEmpty(this.payload.size)) {
+			//[ST]- Fix for the imageSet issues, when the size is undefined
```
[the remark this answers is not in the document]
Comment (Surezt22) [reply source/reactnative/src/components/elements/image.js]: Fixed

**Pull request merged**